config.properties.template

```diff
@@ -206,6 +220,18 @@ wait_chance=0.0
 wait_time_min=0
 wait_time_max=0
 
+# Bot timeouts, to stop the bot after reaching certain definable limits
+# If multiple limits are set the first reached will stop the bot (however it will NOT kill the process, you must do that manually!!!)
+# Set if you want to pause the bot after a certain time (in minutes), if not leave it -1. Only full numers, eg. 300
```

Comment:
This description is a bit contradictionary with the line above (pause the bot <-> stop the bot)

(same with the following 2 descriptions)